### PR TITLE
Added the option to close tabs to the right

### PR DESCRIPTION
**Pull request by d2dyno1**
For #260

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@d2dyno1 Can you disable the close tabs to right menu item when the user right clicks on the right-most tab?

**Review by yair100 — CHANGES_REQUESTED** [review]

**Review by d2dyno1 — COMMENTED** [review]
Added suggestions

File: Files/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -188,6 +188,23 @@ public static async void DuplicateTabAtIndex(object sender, RoutedEventArgs e)
             }
         }
 
+        public static async void CloseTabsToRight(object sender, RoutedEventArgs e)
```
Comment (yaichenbaum) [inline Files/Views/MainPage.xaml.cs:191]:
```suggestion
        public static async void CloseTabsToTheRight(object sender, RoutedEventArgs e)
```

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 2]
```diff
@@ -48,6 +48,14 @@
                                 <FontIcon Glyph="&#xE8A7;" />
                             </MenuFlyoutItem.Icon>
                         </MenuFlyoutItem>
+                        <MenuFlyoutItem
+                            x:Uid="HorizontalMultitaskingControlCloseTabsToTheRight"
+                            Click="{x:Bind views:MainPage.CloseTabsToRight}"
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:53]:
```suggestion
                            Click="{x:Bind views:MainPage.CloseTabsToTheRight}"
```

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 3]
```diff
@@ -48,6 +48,14 @@
...
+                            Text="Close tabs to the right">
+                            <MenuFlyoutItem.Icon>
+                                <FontIcon FontFamily="{StaticResource OldFluentUIGlyphs}" Glyph="&#xE97D;" />
+                            </MenuFlyoutItem.Icon>
+                        </MenuFlyoutItem>
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:58]:
```suggestion
                            Text="Close tabs to the right"/>
```

**Review by yair100 — CHANGES_REQUESTED** [review]

**Comment by d2dyno1** [comment]
@yair100 :+1:

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml (hunk 3 above)
Comment (d2dyno1) [reply Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml]: Should I remove the icon? @yair100

Comment (yair100) [reply Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml]: > Should I remove the icon? @yair100

Yes, please.

Comment (d2dyno1) [reply Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml]: > > Should I remove the icon? @yair100
> 
> Yes, please.

It's gone, reduced to atoms.

**Review by yair100 — APPROVED** [review]
Great work! LGTM

**Review by kal1ms — APPROVED** [review]

**Pull request merged**